### PR TITLE
Use a unique state for each scheduler

### PR DESCRIPTION
**Pull request by andrey-churkin**

### CODE REVIEW

File: tests/composite/test_composite_scheduler.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        for scheduler in schedulers:
+            composite_scheduler.add(scheduler)
+        return composite_scheduler
+
+    expected = _create_composite_scheduler([1, 2])
+    actual = _create_composite_scheduler([0, 0])
+    actual.load_state_dict(expected.state_dict())
+
+    for expected_child, actual_child in zip(expected.child_schedulers, actual.child_schedulers):
+        assert expected_child.delta == actual_child.delta
```
Comment (ljaljushkin) [inline tests/composite/test_composite_scheduler.py:42]: Did this test fail before your fix?

Comment (andrey-churkin) [reply tests/composite/test_composite_scheduler.py:42]: Yes. The reason is that we use one common dictionary for parameters of child schedulers. It is okay for `current_step` or `current_epoch` because it should be the same for all schedulers. But there is another possible case. For example, two schedulers have their own state parameters but these parameters have the same name. These parameters can have different values. The current approach does not work properly in this case.

Also, all child schedulers contain all state parameters as attributes (not just the ones required by them) after calling `PTCompositeCompressionScheduler.load_state_dict()` method in the current approach.

**Review by ljaljushkin — APPROVED** [review]

**Comment by andrey-churkin** [comment]
Jenkins please retry a build

**Comment by alexsu52** [comment]
@andrey-churkin Could you check that the current checkpoints is still loading?

**Comment by andrey-churkin** [comment]
I moved these changes to https://github.com/openvinotoolkit/nncf/pull/511 because both PRs require checkpoints update. I close this PR.
